settings.gradle

```
@@ -14,7 +14,7 @@ gradle.ext.apiVersionFull = '1.16.5-R0.1-SNAPSHOT'
   Every new update should bump the version below according to
   the conventions.
 */
-gradle.ext.version = '1.5.0'
+gradle.ext.version = '1.5.1'
```

You are adding a feature, so the version should be `1.6.0`
```suggestion
gradle.ext.version = '1.6.0'
```